modules/product/src/test/java/com/opengamma/strata/product/fx/FxSingleTest.java

```diff
@@ -158,9 +189,8 @@ public void test_builder_rightOrder() {
 
   public void test_builder_switchOrder() {
     FxSingle test = FxSingle.meta().builder()
-        .set(FxSingle.meta().baseCurrencyAmount(), USD_M1600)
-        .set(FxSingle.meta().counterCurrencyAmount(), GBP_P1000)
-        .set(FxSingle.meta().paymentDate(), DATE_2015_06_30)
```

Comment:
This hasn't changed from before, but I wouldn't mind an explanation of how we end up with the same baseCurrencyAmount when we appear to set it to something different!

Reply:
The currency pair is GBP/USD, so the facyory method needs to swap these around to get GBP as the base and USD as the counter